packages/yoga/src/Result/native/Result.jsx

```diff
@@ -72,7 +70,7 @@ const Result = ({
   <StyledBox divided={divided} display="flex" flexDirection="row">
     {Avatar && <>{isValidElement(Avatar) ? Avatar : <Avatar />}</>}
     <Content>
-      <Attendances attendances={attendances} rate={rate} />
+      {attendances && <Attendances attendances={attendances} rate={rate} />}
```

Comment:
What do you think about supporting an empty list as well? This would be useful in case the attendance list is dynamic

```suggestion
      {!!attendances?.length && <Attendances attendances={attendances} rate={rate} />}
```